runtime/ms_rest_azure/lib/ms_rest_azure/resource.rb

```diff
@@ -23,6 +23,9 @@ class Resource
     # @return [Hash{String => String}] the tags attached to resources (optional).
     attr_accessor :tags
 
+    # @return [String] the resource group of the resource.
+    attr_accessor :resource_group

```

Comment:
should we call it resource_group_name? and update the description to indicate this is the name of the resource group?